docs/src/attribute.md

```diff
@@ -125,7 +125,7 @@ desirable for use inside an attribute value.
     struct Custom data::String end
 
     @htl "<tag att=$(Custom("A&B"))/>"
-    #-> <tag att='…Custom(&quot;A&amp;B&quot;)'/>
```

Comment:
Julia's default show method for a struct will omit the module if it is being rendered inside that same module. Before this PR, this information was not available, and it showed the full name with module prefix.